graph/telemetry/istio/appender/ambient.go

```diff
@@ -66,5 +66,15 @@ func (a AmbientAppender) handleWaypoints(trafficMap graph.TrafficMap, globalInfo
 				}
 			}
 		}
+
+		if remove {
+			graphEdge := []*graph.Edge{}
+			for _, edge := range n.Edges {
+				if edge.Dest.App != WaypointSuffix && edge.Source.App != WaypointSuffix {
```

Comment:
```suggestion
				if !strings.HasSuffix(edge.Dest.App, WaypointSuffix) && !strings.HasSuffix(edge.Source.App, WaypointSuffix) {
```
Not sure, but wondering if equality is correct here...